template-sample-app/template/src/components/common/ZoomControl.js

```diff
@@ -0,0 +1,63 @@
+import React, { useCallback } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Button, ButtonGroup, makeStyles, SvgIcon } from '@material-ui/core';
+import { setViewState } from '@carto/react/redux';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& .MuiButton-contained': {
+      maxWidth: theme.spacing(4.5),
+      backgroundColor: theme.palette.common.white,
+      borderRightColor: theme.palette.other.divider,
+
+      '&.Mui-disabled': {
+        ...theme.typography.caption,
+        color: theme.palette.text.secondary,
+        borderRightColor: theme.palette.other.divider,
+      },
+    },
+  },
+}));
+
+const PlusIcon = (props) => (
+  <SvgIcon width='24' height='24' viewBox='0 0 24 24' className={props.className}>
+    <path d='m19 13h-6v6h-2v-6h-6v-2h6v-6h2v6h6z' fill='inherit' />
+  </SvgIcon>
+);
+
+const MinusIcon = (props) => (
+  <SvgIcon width='24' height='24' viewBox='0 0 24 24' className={props.className}>
+    <path d='m19 13h-14v-2h14z' fill='inherit' />
+  </SvgIcon>
+);
+
+export function ZoomControl(props) {
+  const dispatch = useDispatch();
+  const zoomLevel = useSelector((state) => Math.floor(state.carto.viewState.zoom));
+  const classes = useStyles();
+
+  const increaseZoom = useCallback(() => {
+    dispatch(setViewState({ zoom: zoomLevel + 1 }));
```

Comment:
We're displaying zoomLevel without decimals, but there are fractional zooms indeed, so we might need to round this... thoughts?

Reply:
We have talked during the daily about this, and we decided to not display the 'number'. So just (-) and (+) are required

Reply:
Previously I've added a `Math.floor` to show only for example `3` when the zoom is `3.54345`, but as is decided it was removed in the new layout design task